inference-engine/src/mkldnn_plugin/nodes/mkldnn_roi_pooling_node.cpp

```diff
@@ -71,13 +86,18 @@ struct jit_uni_roi_pooling_kernel_f32 : public jit_uni_roi_pooling_kernel, publi
         L(exit_label);
 
         this->postamble();
+
+        load_emitter->emit_data();
+        if (!mayiuse(avx512_core_bf16) && mayiuse(avx512_core) && store_emitter != nullptr && store_emitter->get_emu_vcvtneps2bf16() != nullptr)
+            store_emitter->get_emu_vcvtneps2bf16()->emit_data();
```

Comment:
Thats just looks weird. I understand that data emittion for BF16 emulator is done in the same way as in MVN, but still seems like it should be encapsulated into store_emitter->emit_data(); call. Anyway it might fixed not in bounds of this PR.

Reply:
I agree, actually
store_emitter->emit_data(); 
Was my initial implementation and this was the reason of failed CI.
For some reason bf16 logic is not covered in store_emitter->emit_data();